### PR TITLE
[#183] Added primary ML endpoint to readme.

### PR DESCRIPTION
**Pull request by arekmano**
[#183]

- [x] Added primary ML endpoint to readme

### CODE REVIEW

File: src/ml_service/README.md [hunk 1]
```diff
@@ -15,6 +15,127 @@
...
+**Method** : `POST`
+
+**Data constraints**
+
+Provide facts_vector and demands_vector, with key values for each fact/demand.
+
+```json
+{
+  "demands" : {
+    "demand_lease_modification": 1,
```
Comment (Samuel-Campbell) [inline src/ml_service/README.md:79]: why are all the values 1?

**Comment by dodoels** [comment]
@arekmano it is a bit confusing if you have all `1` as the value, can you change some to `0` instead

File: src/ml_service/README.md (hunk 1 above)
Comment (arekmano) [reply src/ml_service/README.md:79]: dummy example

Comment (arekmano) [reply src/ml_service/README.md:79]: updated

**Review by dodoels — APPROVED** [review]

**Comment by lancelafontaine** [comment]
This is only one of the many endpoints that would be required for integration with `nlp_service` right? `nlp_service` will (eventually) need a way to know what fact to ask the user next.

**Comment by arekmano** [comment]
yes. This is the initial api that we can demo this iteration. we will later need an next_fact endpoint or similar

**Review by ghost — CHANGES_REQUESTED** [review]
Code Cov bug...?

**Review by ghost — APPROVED** [review]

**Pull request merged**